### PR TITLE
fixes moths/arachnids/dionae/slimes not having burn visuals

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
Also changes all species melee weapons to be a fist

Gives all species their blood reagent

## Why / Balance
Fists: All species now deal blunt damage, so a lizard clawing when they deal blunt doesn't make sense
Helmet change: Looks better

also fixes moths/arachnids/dionae/slimes not having burn visuals

## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
No

### CODE REVIEW

**Comment by LankLTE** [comment]
I think this is kind of a bummer, I like the antennae being visible

**Comment by Tunguso4ka** [comment]
We had standard punch for lizards before the playtests, but i asked Smug/Whisper to leave clawing visuals.

I like them, they look nice.

It kinda brraks my mind when im punching as lizard/unathi and see standard punch visual instead of clawing one.

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
I actually like the helmet not hiding head cosmetics, but on the topic of damage visuals, I originally wanted the species blood to be remade to just be a child of blood with different reagent colors, which is why it was left in the species.

**Comment by P-MCT** [comment]
I'd have to agree on Tunguso on their comment.

Slash/Blunt damage is considered brute, but if you slash with machete for example, it's considered brute damage. I don't see why you wouldn't allow those with claws to keep their visuals. Like the races itself, it's a cosmetic choice that differentiates from standard.

In regards to the helmets, again this is a matter of personal opinion, I prefer the helmets hiding head related visuals, but many won't. Many liz mains for example, pride their unique design and might frown on this choice, with their horns being hidden, or hiding their glowing fin ray.

**Comment by Dutch-VanDerLinde** [comment]
> I actually like the helmet not hiding head cosmetics, but on the topic of damage visuals, I originally wanted the species blood to be remade to just be a child of blood with different reagent colors, which is why it was left in the species.

Do blood pack s still work with differen t blood reagents

**Comment by Dutch-VanDerLinde** [comment]
> I'd have to agree on Tunguso on their comment.
> 
> Slash/Blunt damage is considered brute, but if you slash with machete for example, it's considered brute damage. I don't see why you wouldn't allow those with claws to keep their visuals. Like the races itself, it's a cosmetic choice that differentiates from standard.
> 
> In regards to the helmets, again this is a matter of personal opinion, I prefer the helmets hiding head related visuals, but many won't. Many liz mains for example, pride their unique design and might frown on this choice, with their horns being hidden, or hiding their glowing fin ray.

Idk I just made this pr cuz i wanna punch people as a lizard

**Comment by Vermidia** [comment]
I just realized Diona's here had blue blood. It should be orange because water -> sap blood happened upstream

**Comment by Dutch-VanDerLinde** [comment]
> As a diona player I feel red blood visuals are gonna look really out of place. They used to be light blue, now they are light orange, but a heavy red is a stark contrast.

Agreed but its just for consistency(if i can i'll do the thing that whisper said)

**Comment by Rainbeon** [comment]
As a moth player I would also really miss the antennae being visible + the insect blood, species are essentially just reskins of humans in this fork for visual flavor, and removing more of their visual flavor removes a lot of reason to play them in the first place

I like the burn visual change, however

**Comment by Kadeo64** [comment]
I wonder if it's possible to hide human hair, I think that's a bigger source of ugliness than visible horns/antennae

**Comment by Dutch-VanDerLinde** [comment]
> I wonder if it's possible to hide human hair, I think that's a bigger source of ugliness than visible horns/antennae

needs something more advanced

**Comment by Kadeo64** [comment]
> needs something more advanced

did we not have a hidehair system for hardsuit helmets at some point

**Comment by Dutch-VanDerLinde** [comment]
> > needs something more advanced
> 
> did we not have a hidehair system for hardsuit helmets at some point

thats hide the entire hair; for helmets, you'd want people with longhair to still appear but not clip through the helmet

**Comment by Dutch-VanDerLinde** [comment]
changes:
damage visuals are back to normal, species now have their unique blood reagents (purely cosmetic, they all inject into every other species fine)

**Comment by QuietlyWhisper** [comment]
below pr is preferred for helmet hiding stuff
- #3600

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/moth.yml [hunk 1]
```diff
@@ -5,8 +5,13 @@
   name: Urist C.M McFluff
   suffix: RMC14
   components:
+  - type: Bloodstream
+    bloodReagent: RMCInsectBlood
   - type: HumanoidAppearance
     species: Moth
+    hideLayersOnEquip:
+    - Snout
+    - HeadTop
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Mobs/Species/moth.yml:14]:
```suggestion
```

File: Resources/Prototypes/_RMC14/Entities/Mobs/Species/slime.yml [hunk 2]
```diff
@@ -5,6 +5,8 @@
   name: Urist C.M McSlime
   suffix: RMC14
   components:
+  - type: Bloodstream
+    bloodReagent: RMCSlimeBlood
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Mobs/Species/slime.yml:9]:
```suggestion
```

**Review by QuietlyWhisper — COMMENTED** [review]
Smug told me it would be better to leave the blood reagents for a blood type implementation, thanks for the pr

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**